packages/graphback-runtime-knex/tests/service/DefaultCRUDServiceTest.ts

```diff
@@ -262,94 +244,11 @@ test('find users where name starts with "John"', async () => {
     }
   })
 
-  const result = await services.User.findBy({ name: { startsWith: 'John' } }, { graphback: { services: {}, options: { selectedFields: ["id"] } } })
+  const result = await services.User.findBy({ filter: { name: { startsWith: 'John' } } })
 
   expect(result.items).toHaveLength(2)
 })
 
-test('find and count all users', async () => {
-  const { services } = await setup({
-    schemaSDL: `
-    """
-    @model
```

Comment:
I suppose this block is deleted because it was hard to fake an `info` object?

Reply:
Yes, that is an unfortunate compromise. I tried for some time yesterday to create a mock info object but to no avail, and to my knowledge there is no library to do it.

To make up for the loss of this tested I added an integration test: https://github.com/aerogear/graphback/blob/6a6deec2c5189456ab08c2fd10f6315ec2dd26fe/integration/tests/runtime-workflow-postgres.ts#L305

Reply:
Perfect, I noticed those. We can live with that. Thanks for checking it out.